src/qwed_new/core/fact_verifier.py

```
@@ -107,106 +108,134 @@ def verify_fact(
+            # Step 7: Calculate deterministic verdict
+            verdict, confidence, reasoning = self._calculate_verdict(
+                semantic_score=semantic_score,
+                keyword_score=keyword_score,
+                entity_match=entity_match,
+                has_negation=has_negation,
+                citations=citations,
+                keyword_details=keyword_details,
+                entity_details=entity_details,
+                negation_details=negation_details
+            )
+        except Exception as exc:
```

Ruff (0.16.0)

[warning] Do not catch blind exception: `Exception`

(BLE001)